modules/tinymce/src/plugins/table/main/ts/ui/CellDialogGeneralTab.ts

```diff
@@ -12,18 +12,8 @@ import { Option } from '@ephox/katamari';
 import { Types } from '@ephox/bridge';
 
 const getClassList = (editor: Editor) => {
-  const rowClassList = getCellClassList(editor);
-
-  const classes: Types.SelectBox.ExternalSelectBoxItem[] = Helpers.buildListItems(
-    rowClassList,
-    (item) => {
```

Comment:
this seems to be left over from T4 or T3 or something, since textStyle isn't in the select items spec and isn't referenced anywhere else in the monorepo apart from the couple places its set in the table plugin